### PR TITLE
suthesh/h1-tag-seo

### PR DESCRIPTION
**Pull request by suthesan-deriv**
Changes:

-   replace h1 with span in header

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by yashim-deriv** [comment]
Since it seems that `as` has the value of `span` in the majority of `NavCard` props, isn't it better to just set the default value of `as="span"` in that component itself?

File: src/components/custom/other-platforms.js [hunk 1]
```diff
@@ -237,6 +237,7 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
             <Flex direction="column" wrap="wrap" jc="flex-start">
                 <StyledText>{localize('Trading platforms')}</StyledText>
                 <NavCard
+                    as="span"
```
Comment (sean-binary) [inline src/components/custom/other-platforms.js:240]: I don't think you can use the `as` prop here. You can either add `.attrs({ as: 'span' })` directly onto the global `NavCard` and `CardLink` component, or you could extend those onto this page here and add `.attrs({ as: 'a' })` to it.

Comment (suthesan-deriv) [reply src/components/custom/other-platforms.js:240]: Actually the `as` prop is passed to NavCard -> ResponsiveHeader -> Header . Technically it works but I think it's better to change the prop name to `title_as` cause it's kinda funny to suddenly see an `as` prop in a Card component? 😂

**Comment by suthesan-deriv** [comment]
> Since it seems that `as` has the value of `span` in the majority of `NavCard` props, isn't it better to just set the default value of `as="span"` in that component itself?

I thought of that too actually, in fact that's what I did in my first commit 😂 ...then thought what if we in the future decide to use that NavCard and CardLink component else where? Do you think that might happen? 🤔

**Review by sean-binary — APPROVED** [review]

**Pull request merged**